core/controllers/topics_and_skills_dashboard.py

```diff
@@ -206,7 +206,7 @@ def post(self):
         if (sort_by is not None and
                 not isinstance(sort_by, python_utils.BASESTRING)):
             raise self.InvalidInputException(
-                'Sort by should be a string.')
+                'Sort by value should be a string.')
```

Comment:
Please look carefully at the previous comment -- the idea was to group "sort by". You can put quotes around "sort by" or write sort_by, but otherwise the sentence isn't easy to parse.

Reply:
Updated this to 
```
 'The value of sort_by should be a string.')
```